### PR TITLE
refs sparkfabrik-innovation-team/board#372: fix multiarch process

### PR DESCRIPTION
**Pull request by Monska85**
We need to fix the download of the static binaries to fully support the multiarch process.

### CODE REVIEW

**Comment by paolomainardi** [comment]
@Monska85 is still in Draft or just forgotten to remove it ?

**Comment by Monska85** [comment]
> @Monska85 is still in Draft or just forgotten to remove it ?

Yes @paolomainardi it is still in draft. I need to fix blackfire because it is a static amd64 binary.

**Comment by Monska85** [comment]
@paolomainardi now the PR is ready for review

File: 7.2.0-fpm-alpine3.7/docker-entrypoint.sh [hunk 1]
```diff
@@ -14,6 +14,9 @@ export XDEBUG_ENABLE=${XDEBUG_ENABLE:-0}
 export MAILHOG_HOST=${MAILHOG_HOST:-"mail"}
 export MAILHOG_PORT=${MAILHOG_PORT:-1025}
 
+# Blackfire configurations.
+export BLACKFIRE_APM_ENABLED=${BLACKFIRE_APM_ENABLED:-0}
```
Comment (paolomainardi) [inline 7.2.0-fpm-alpine3.7/docker-entrypoint.sh:18]: should we add this one to all entrypoints ?

File: 7.1.11-fpm-alpine3.4/docker-entrypoint.sh [hunk 2]
```diff
@@ -14,6 +14,9 @@ export XDEBUG_ENABLE=${XDEBUG_ENABLE:-0}
 export MAILHOG_HOST=${MAILHOG_HOST:-"mail"}
 export MAILHOG_PORT=${MAILHOG_PORT:-1025}
 
+# Blackfire configurations.
+export BLACKFIRE_APM_ENABLED=${BLACKFIRE_APM_ENABLED:-0}
```
Comment (paolomainardi) [inline 7.1.11-fpm-alpine3.4/docker-entrypoint.sh:18]: questa riga deve essere riportata su tutti gli entrypoint ?

**Pull request merged**